examples/aggregator/backend/Cargo.toml

```diff
@@ -17,6 +17,7 @@ structopt = "*"
 tokio = { version = "*", features = [
   "fs",
   "macros",
+  "rt-multi-thread",
```

Comment:
From the CI errors, it seems like this might be needed in a few other examples where there are rust clients using tokio.